src/fonts/OTF/TeX/makeFF

```diff
@@ -2172,21 +2172,7 @@ $extra{'Main-Italic'} = {
   diacriticals => [
     'Select(0uB0)',  'SetRBearing(-160,1)',   # \degree
   ],
-};
-
-$extra{'Typewriter'} = {
-  space => [
-    'Select(0u20)','Clear()',
-    'SetRBearing(525)',
-  ],
-
-  spaceNB => [
-    'Select(0uA0)','Clear()',
-    'SetRBearing(525)',
-  ],
-};
 
-$extra{'Math-Italic'} = {
```

Comment:
I combined this 'Math-Iatlic' section with the one right above it.  No need to have two.